gluten-iceberg/src/main/scala/org/apache/gluten/execution/IcebergScanTransformer.scala

```diff
@@ -56,7 +56,7 @@ case class IcebergScanTransformer(
 
   override lazy val fileFormat: ReadFileFormat = GlutenIcebergSourceUtil.getFileFormat(scan)
 
-  override def getSplitInfos: Seq[SplitInfo] = {
+  override def getSplitInfosFromPartitions(partitions: Seq[InputPartition]): Seq[SplitInfo] = {
```

Comment:
@acvictor Can you help review this change?

Reply:
Looks good to me, thank you!